Agilefreaks/FreaksCatalog-API
# [130] Add photo to a freak

## Description
Create a model for Photo and a table 'photos' in database. This model is used to store the url of the photo for each freak and project. Also, here is created a one-to-one relationship between freak and photo, so each freak has one photo and each photo belongs to a single freak

## Related

[[130] Add Photo to a Freak](https://trello.com/c/0yKfo1ER/130-add-photo-to-a-freak)

## Steps to Test or Reproduce
Run test that can be found on `spec/controllers/graphql/queries/freaks_spec.rb`. This test create 2 freak objects and assign for each of them a photo object. You can see the output of this test in `spec/fixtures/responses/queries/freaks`.

Also, you can test functionality of this implementation by running another test that can be found in `spec/models/photo_spec.rb`. This test create a Photo object with a given uri and check if the output is as expected.
